src/mapping/morpho-aave-v3/morpho-aave-v3.ts

```diff
@@ -49,8 +49,8 @@ import {
   ReserveFeeClaimed,
   UserNonceIncremented,
 } from "../../../generated/morpho-v1/MorphoAaveV3/MorphoAaveV3";
-import { Market, UnderlyingTokenMapping } from "../../../generated/morpho-v1/schema";
-import { AAVE_V3_ORACLE_OFFSET, BASE_UNITS, BIGDECIMAL_ONE, RAY_BI } from "../../constants";
+import { EMode, Market, UnderlyingTokenMapping } from "../../../generated/morpho-v1/schema";
+import { AAVE_V3_market_oracle_OFFSET, BASE_UNITS, BIGDECIMAL_ONE, RAY_BI } from "../../constants";
```

Comment:
```suggestion
import { AAVE_V3_MARKET_ORACLE_OFFSET, BASE_UNITS, BIGDECIMAL_ONE, RAY_BI } from "../../constants";
```
I think you replaced with lowercase chars in multiple files by mistake